cns/restserver/util.go

```diff
@@ -756,12 +756,8 @@ func (service *HTTPRestService) SendNCSnapShotPeriodically(ctx context.Context,
 }
 
 func (service *HTTPRestService) validateIPConfigsRequest(ctx context.Context, ipConfigsRequest cns.IPConfigsRequest) (cns.PodInfo, types.ResponseCode, string) {
```

Comment:
i think we can get rid of this function for good. All that it is actually validating now is whether the `ipConfigsRequest.OrchestratorContext` is nil and returning the podInfo, and even then, `cns.NewPodInfoFromIPConfigsRequest` will throw an error if the the orchestrator context is empty anyways. Let's move the getting podInfo part outside to where it is needed.

Reply:
Hi @nddq  I think for some funcs, they need to return errorCode and errorMessage; for example:
requestIPConfigHandlerHelper()